tools/sof-kernel-log-check.sh

```diff
@@ -1,64 +1,70 @@
 #!/bin/bash
 
-begin_line=${1:-1}
+begin_timestamp=${1:"NONE"}
```

Comment:
@plbossart I think this is some bash issue. It should be used as `${[number]:-default}`
Reference https://unix.stackexchange.com/questions/338146/bash-defining-variables-with-var-number-default
```suggestion
begin_timestamp=${1:-NONE}
```

Reply:
ok, thanks for the suggestion, will give it a try.

Reply:
I missed the fact that the code already uses 0 as a possible parameter, so the default should also be zero here and we use the entire log.